content/TrueCommand/Reports/Creation.md

```diff
@@ -14,43 +14,51 @@ The **Reports** page customizes system metrics charts for data analysis.
 Enter a report name and an optional description for the report.
 
 ![ReportsCreate](/images/TrueCommand/2.0/ReportsCreateReportNameDescription.png "Create a new report")
 
-Click *BROWSE WIDGETS* or *WIDGET* to add charts to the report.
+Click **BROWSE WIDGETS** or **WIDGET** to add charts to the report.
 
 ![ReportsCreateWidgetsAdd](/images/TrueCommand/2.0/ReportsCreateReportWidgets.png "Adding widgets to a new report")
 
 ### Custom Charts
 
-Most charts are already configured to report certain settings.
-To create a custom chart with custom settings, add a *Custom Area Chart*, *Custom Bar Chart*, or *Custom Line Chart*.
-Fill in these options when adding a custom chart:
+TrueCommand already configures most charts to report specific settings.
+To create a custom chart with custom settings, add a **Custom Area Chart**, **Custom Bar Chart**, or **Custom Line Chart**.
+Fill in the following options when adding a custom chart:
 
 * General settings: Enter a Title, Subtitle (optional), Axis label (optional), Point size, Line size, Y min (optional), and Y max (optional) for the chart.
-  `Stack the values` can be set to bring data points on the chart closer together.
-  This setting is useful for charts that have many different data points at the max Y value.
-* Data sources: Add data sources to the chart by expanding a category and selecting appropriate sources.
-  Multiple data sources can be added to one chart.
-* Summary: This step shows the all of the chosen values.
-  Click *SAVE* to add the custom chart to the report or *BACK* to go back and change a setting or data source.
+  You can set `Stack the values` to bring data points on the chart closer together.
+  `Stack the values` is helpful for charts with many different data points at the max Y value.
+* Data sources: Add data sources to the chart by expanding a category and selecting which sources you want.
+  You can add multiple data sources to one chart.
+* Summary: This step shows all chosen values.
+  Click **SAVE** to add the custom chart to the report, or **BACK** to change a setting or data source.
 
 After adding charts to the report, click **SAVE** to make this report available for use.
 
 
 ![ReportsCreateReportSave](/images/TrueCommand/2.0/ReportsCreateReportSave.png "Reports Create Report Save")
 
-After creating a report, you can click **GENERATE** to [generate the report]({{< relref "Generation.md" >}}) or you can go back to the reports page and make create another report.
+After creating a report, you can click **GENERATE** to [generate the report]({{< relref "Generation.md" >}}), or you can go back to the reports page and create another.
 
 ![ReportsGenerate](/images/TrueCommand/2.0/ReportsGenerate.png "Reports Generate")
 
+## Edit a Report
+
+Open the Reports page and click <i class="material-icons" aria-hidden="true" title="edit">edit</i> next to the report name to rename a report, add a description, or add/delete a Widget.
```

Comment:
Suggest making Reports bold as it refers to the specific screen and name of the screen.